src/Pages/SignUp/SignUp.js

```diff
@@ -1,9 +1,270 @@
-import React, { Component } from "react";
-
+import React, { Component } from 'react';
+import CheckList from './CheckList';
+import './SignUp.scss';
+import {signupAPI,smsAPI,checkAPI} from "../../config";
 class SignUp extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name_box: '',
+      number: '',
+      id: '',
+      pw: '',
+      confirm: '',
+      confirmNum: true,
+      pw_again: '',
+      pwAlert: true,
+      numberValid: true,
+      allChecked: false,
+      checked0: false,
+      checked1: true,
+      checked2: false,
+      checkList: [],
+    };
+  }
+
+  handleInputValue = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+    const { number } = this.state;
+    const checkNumber = !isNaN(number) && number.length >= 9;
+
+    this.setState({
+      numberValid: checkNumber? false: true
```

Comment:
템플릿에 질문주신 내용에 대해 답변 드리도록 하겠습니다. 
checkNumber에 할당된 조건이 true 이면 불리안 값이 false로, false이면 true로 변경하신 코드이네요.
그러면 checkNumber의 불리안값의 반대 불리안값이 this.state.numberValid에 담기는 거겠죠?
반대 불리안 값을 할당하기 위해 아래와 같이 '!'를 이용하시면 되겠습니다.
```suggestion
      numberValid: !checkNumber
```